packages/react-components/react-shared-contexts/src/OverridesContext/OverridesContext.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+/**
+ * @internal
+ */
+export type OverridesContextValue = {
+  // should be InputProps['appearance'], but do we want to add the dependency?
+  inputDefaultAppearance?: 'outline' | 'underline' | 'filled-darker' | 'filled-lighter';
```

Comment:
```suggestion
  inputAppearance?: 'outline' | 'underline' | 'filled-darker' | 'filled-lighter';
```

Do "default" add any meaning there?

Reply:
This options sets the default `inputAppearance`, does not overwrite anything:
```tsx
<FluentProvider overrides_unstable={{inputDefaultAppearance: 'filled-darker'}}>
  <Input /> /* 👉 appearance is filled-darker from overrides context */
  <Input appearance="underline" /> /* 👉 appearance is underline from props */
</FluentProvider>
```

For that reason I used the 'default' word.

Reply:
Ok, from that perspective it makes sense 👍